tgui/packages/tgui/interfaces/LoreConsole.tsx

```diff
@@ -0,0 +1,61 @@
+import { useBackend } from '../backend';
+import { useState } from 'react';
+import { Box, Button, Section, Stack } from 'tgui-core/components';
+import { Window } from '../layouts';
+import { processedText } from '../process';
+
+type LoreEntry = {
+  title: string;
+  body: string;
+};
+
+type LoreConsoleData = {
+  entries: LoreEntry[];
+};
+
+export const LoreConsole = (props) => {
+  const { data } = useBackend<LoreConsoleData>();
+  const { entries } = data;
+  const [currentPage, setCurrentPage] = useState(0);
+  const entry = entries[currentPage];
+  return (
+    <Window width={700} height={700} theme="amberpos">
+      <Stack fill vertical>
```

Comment:
This needs some padding around it, the text is very close to the edges of the window